packages/gestalt/src/Masonry.js

```diff
@@ -325,18 +361,110 @@ export default class Masonry<T> extends React.Component<Props<T>, State<T>> {
     this.scrollContainer = ref;
   };
 
-  fetchMore = () => {
-    const { loadItems } = this.props;
-    if (loadItems && typeof loadItems === 'function') {
-      this.setState(
-        {
-          isFetching: true,
-        },
-        () => loadItems({ from: this.props.items.length })
-      );
+  getLayout = () => {
```

Comment:
As a style nit - these functions (`getLayout`, `calculateRenderLayout`) should probably sit outside of the class as helper functions. The words "get" (a function that returns something) and "calculate" (a function that returns something) are smells of this to me. Advantages are that dependencies on props vs. state can be clearly factored away.

Reply:
done. moved to sit outside the class as helpers and renamed.